Ira/my-app-video/src/components/searchResultSection/SearchResultSection.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import {Button} from "../shared/button/Button";
+import './styles.scss';
+
+interface SearchResultSectionProps {
+  stateSortByBtn: string,
+  result: number,
+  sortByParam: string,
+  searchValue: string,
```

Comment:
This component shouldn't know about title **searchValue**,  **searchBy**, **stateSortByBtn** , **history**.  You've overloaded your component with these props and made it hard to mantaince and reuse. This component just has to render toggle, number of movies and call some click handler on click. Thats it! **_Alway think of how to make reusable components simple and make its logic clear._**